bash/atomic

```diff
@@ -578,7 +578,7 @@ _atomic_containers_trim() {
 _atomic_containers_update() {
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--help --set" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--help --set --to" -- "$cur" ) )
```

Comment:
I wonder if we should ever make these smarter, since we don't display the options for "set" and "to" when you try to continue auto complete (unlike, say, the --storage flag for pull). But that would be somewhat difficult.